content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst

```diff
@@ -0,0 +1,244 @@
+localization:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Egypt - Accounting`
+     - ``l10n_eg``
+     - Default :doc:`fiscal localization package <../overview/fiscal_localization_packages>`
+   * - :guilabel:`Egyptian E-invoice Integration`
+     - ``l10n_eg_edi_eta``
+     - :ref:`Egyptian Tax Authority (ETA) e-invoicing integration <egypt/e-invoicing>`
+
+.. _egypt/e-invoicing:
+
+Egyptian e-invoicing
+====================
+
+Odoo is compliant with the **Egyptian Tax Authority (ETA) e-invoicing** requirements.
+
+.. important::
+   Egyptian e-invoicing is available from Odoo 15.0. If needed, :doc:`upgrade
+   </administration/upgrade>` your database.
+
+.. seealso::
+   - `Video: Egypt E-invoicing <https://www.youtube.com/watch?v=NXuBPLR4pVw>`_
+   - :doc:`/administration/upgrade`
+
+.. _egypt/e-invoicing-eta-portal:
+
+Register Odoo on your ETA portal
+--------------------------------
+
+You must register your Odoo ERP system on your ETA portal to get your API credentials. You need
+these codes to :ref:`configure your Odoo Accounting app <egypt/e-invoicing-configuration>`.
+
+Access your company profile on the ETA portal by clicking on :guilabel:`View Taxpayer Profile`.
+
+.. image:: egypt/taxpayer-profile.png
+   :align: center
+   :alt: Clicking on "View Taxpayer Profile" on an ETA invoicing portal
+
+Next, go to the :guilabel:`Representatives` section and then click on :guilabel:`Register ERP`.
+Fill out the :guilabel:`ERP Name` (e.g., ``Odoo``) and leave the other fields empty.
+
+.. image:: egypt/add-erp-system.png
+   :align: center
+   :alt: Filling out of the form to register an ERP system on the ETA portal.
+
+Once successfully registered, the website displays your API credentials:
+
+- Client ID
+- Client Secret 1
+- Client Secret 2
+
+.. note::
+   - ETA should give you a username and a password to access their online portal.
+   - Ask ETA to provide you with preproduction portal access as well.
+   - These codes are confidential and should be stored safely.
+
+.. _egypt/e-invoicing-configuration:
+
+Configuration on Odoo
+---------------------
+
+To connect your Odoo database to your ETA portal account, go to :menuselection:`Accounting -->
+Configuration --> Settings --> ETA E-Invoicing Settings`, and set the :guilabel:`ETA Client ID` and
+:guilabel:`ETA Secret` that you retrieved when you :ref:`registered Odoo on your ETA portal
+<egypt/e-invoicing-eta-portal>`. Set an invoicing threshold if needed.
+
+.. image:: egypt/eta-api-integration.png
+   :align: center
+   :alt: Configuration of the ETA E-Invoicing credentials in Odoo Accounting
+
+.. important::
+   - **Test on your preproduction portal** before starting to issue real invoices on the production
+     ETA portal.
+
+     - **Credentials** for preproduction and production environments are different. Make sure to
```

Comment:
Looks like this line is over-indented?

Reply:
It wasn't but it can be up a level if you prefer.